File: testDataOutput/13_dataClass.swift
public class User{
    let name: String

    var address: String

    var age: Int

    let admin: Bool

    init( name: String, address: String, age: Int, admin: Bool) {

        self.name = name
        self.address = address
        self.age = age
        self.admin = admin
    }
}

func main(args: [String]) {
    let bob = User(name : "bob", address : "london", age : 45, admin : false)
    print(bob.toString())
    bob.age += 3
    print(bob.toString())
}

File: testData/13_dataClass.kt
package com.test

data class User(val name: String = "", var address: String = "", var age: Int = 52, val admin: Boolean): Codable

fun main(args: Array<String>) {
    val bob = User(name = "bob", address = "london", age = 45, admin = false)
    println(bob.toString())
    bob.age += 3
    println(bob.toString())
}
